src/components/Result.vue

```diff
@@ -47,6 +47,18 @@ export default {
       });
       this.totalResponses = result.length;
     },
+    ResultsDisplayed(val) {
+      let CorrectCount = 0
+      for (let i=0;i<val.length;i++)
+      {
+        if (val[i]["selectedOption"] === val[i]["correct_answer"])
+        {
+          CorrectCount+=1
+        }
+      }
+      window.alert("You gave " + CorrectCount + " Correct Answers");
+    }
+
```

Comment:
@gitshashwat 

1. Please replace the logic with ES6 loops 
> Refer: https://hacks.mozilla.org/2015/04/es6-in-depth-iterators-and-the-for-of-loop/

2. Please view the correct answer inside the bottom sheet used for the Result.
> Refer: https://vuetifyjs.com/en/components/bottom-sheets/